packages/fluentui/react-northstar/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -155,19 +155,20 @@ const DropdownSelectedItem: React.FC<WithAsProp<DropdownSelectedItemProps>> &
           }),
       };
 
-  const imageProps = _.isNil(image)
-    ? image
-    : {
-        children: (ComponentType, props) =>
-          Image.create(image, {
-            defaultProps: () => ({
-              avatar: true,
-              className: dropdownSelectedItemSlotClassNames.image,
-              styles: resolvedStyles.image,
+  const imageProps =
+    _.isNil(image) || image?.hasOwnProperty('children')
```

Comment:
I tried in previously commit to solve it using childrenExist but it will be true for text only and then not working as expected. If you don't agree with this solution, have any other idea? @layershifter and @pompomon

Reply:
Expected fix there is to remove `Label` component from there like it was done with `DropdownItem`

https://github.com/microsoft/fluent-ui-react/pull/2382/

Reply:
In this case there is no need to call factories for `imageProps`

Reply:
Done